### PR TITLE
docs(pages): Adding TOC and several mdbook plugins

### PR DESCRIPTION
**Pull request by grzesuav**

### CODE REVIEW

**Comment by grzesuav** [comment]
TOC plugin : 
image

**Comment by grzesuav** [comment]
found on github - at the bottom at the page - 
image

File: docs/src/contrib/build.md [hunk 1]
```diff
@@ -66,6 +68,23 @@ go get -u k8s.io/code-generator/cmd/{lister,client,informer,deepcopy}-gen
...
 ```
 
+## Documentation build
+
+Documentation is generated from `.md` files with [mdbook](https://github.com/rust-lang/mdBook).
+To generate documentation, you need to install:
+* mdbook
+* mdbook plugins:
+    * [linkcheck](https://crates.io/crates/mdbook-linkcheck) - verifies link corectness
+    * [toc](https://crates.io/crates/mdbook-linkcheck) - creates TOC's - table of content
```
Comment (niladrih) [inline docs/src/contrib/build.md:78]:
```suggestion
    * [toc](https://crates.io/crates/mdbook-toc) - creates TOC's - table of content
```

**Review by niladrih — APPROVED** [review]
LGTM

**Pull request merged**